src/daemon.js

```diff
@@ -122,6 +132,9 @@ class Daemon {
       }
 
       const daemon = execa(this._binPath, execOptions)
+      if (process.env.DEBUG) {
+        daemon.stderr.pipe(process.stderr)
+      }
```

Comment:
I didnt update this in the other PR yet, but this conditional isn't needed.
```suggestion
      daemon.stderr.pipe(process.stderr)
```